### PR TITLE
Add coverage for `@ConfigMapping` annotation

### PR DESCRIPTION
**Pull request by Sgitario**
Examples are inspired by https://github.com/smallrye/smallrye-config/blob/main/cdi/src/test/java/io/smallrye/config/inject/ConfigMappingInjectionTest.java
More info about this annotation: https://smallrye.io/docs/smallrye-config/mapping/mapping.html

### CODE REVIEW

**Comment by Sgitario** [comment]
@kshpak would you mind to take a look into this PR? Thanks in advance,

**Comment by kshpak** [comment]
I will take a look in 30 mins

**Comment by kshpak** [comment]
The test is failing with latest quarkus main. Apparently, io.smallrye.config expect that all properties with prefix "server" will be used

**Comment by Sgitario** [comment]
> The test is failing with latest quarkus main. Apparently, io.smallrye.config expect that all properties with prefix "server" will be used

Fixed. Would you mind taking another look?

File: 022-quarkus-properties-config-all/src/main/java/io/quarkus/qe/configmapping/Person.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    int age;
+
+    public String getName() {
+        return name;
+    }
+
+    public int getAge() {
+        return age;
+    }
+}
```
Comment (kshpak) [inline 022-quarkus-properties-config-all/src/main/java/io/quarkus/qe/configmapping/Person.java:17]: getters can be deleted and a field can be accepted directly by calling a variable name. IMO this is the added value of config class vs. config interface

Comment (kshpak) [reply 022-quarkus-properties-config-all/src/main/java/io/quarkus/qe/configmapping/Person.java:17]: Now I realize that the additional logic can be placed under getters. This is up to you now.

**Review by kshpak — APPROVED** [review]

File: 022-quarkus-properties-config-all/src/main/java/io/quarkus/qe/configmapping/Person.java (hunk 1 above)
Comment (Sgitario) [reply 022-quarkus-properties-config-all/src/main/java/io/quarkus/qe/configmapping/Person.java]: Updated

**Review by kshpak — APPROVED** [review]

**Pull request merged**